src/services/rariPoolsAPY.js

```diff
@@ -68,21 +69,27 @@ export const getAaveApyBNs = async () => {
   /* eslint-disable i18next/no-literal-string */
   const query = `{
     reserves(where: {
-      symbol_in: ["DAI", "USDC", "USDT", "TUSD", "BUSD", "SUSD"]
+      symbol_in: ["DAI", "USDC", "USDT", "TUSD", "BUSD", "SUSD", "ETH"]
     }) {
+      id
       symbol
       liquidityRate
     }
   }`;
   /* eslint-enable i18next/no-literal-string */
   const data = await callSubgraph(getEnv().AAVE_SUBGRAPH_NAME, query);
   if (!data) return null;
-  const result = data.reserves.reduce((APY, reserve) => {
+  const result = data.reserves
+    .filter(reserve => {
+      return reserve.symbol !== ETH ||
+        reserve.id === '0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee0x24a42fd28c976a61df5d00d0599c34c4f90748c8';
```

Comment:
Maybe this could be extracted to a named constant? I'm suggesting mainly because it's not quite clear why this particular id is special, so a comment would be fine, too.